.git-blame-ignore-revs

```diff
@@ -1 +1,2 @@
 9d1fd093071278be77428dba189d958d5b7a7aeb
+0e7be75ad2a8e2443894b84c6e9c377f9819125e
```

Comment:
Again, beware of the required update when you rebase.

Reply:
Thanks, I've just updated the hash after rebase.